pkg/clients/redis/redis.go

```diff
@@ -19,35 +19,38 @@ package redis
 import (
 	"context"
 	"encoding/json"
-	"fmt"
 	"reflect"
 
-	redismgmt "github.com/Azure/azure-sdk-for-go/services/redis/mgmt/2018-03-01/redis"
+	"github.com/Azure/azure-sdk-for-go/services/redis/mgmt/2018-03-01/redis"
 	"github.com/Azure/azure-sdk-for-go/services/redis/mgmt/2018-03-01/redis/redisapi"
 	"github.com/Azure/go-autorest/autorest/azure/auth"
 	"github.com/pkg/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	azure "github.com/crossplaneio/stack-azure/pkg/clients"
 
 	"github.com/crossplaneio/stack-azure/apis/cache/v1alpha3"
+	azure "github.com/crossplaneio/stack-azure/pkg/clients"
 )
 
-// NamePrefix is the prefix for all created Azure Cache instances.
-const NamePrefix = "acr"
+// Resource states
+const (
+	ProvisioningStateCreating  = string(redis.Creating)
+	ProvisioningStateDeleting  = string(redis.Deleting)
+	ProvisioningStateFailed    = string(redis.Failed)
+	ProvisioningStateSucceeded = string(redis.Succeeded)
+	ProvisioningStateUpdating  = string(redis.Updating)
+)
 
 // A Client handles CRUD operations for Azure Cache resources. This interface is
 // compatible with the upstream Azure redis client.
 type Client redisapi.ClientAPI
```

Comment:
It looks like this can be removed now that we are using `redisapi.ClientAPI` directly.